src/movepick.h

```diff
@@ -118,7 +111,8 @@ class MovePicker {
   ExtMove* end() { return endMoves; }
 
   const Position& pos;
-  const Histories* ht;
+  const ButterflyHistory* mainHistory;
+  const PieceToHistory** contHistory;
   Move ttMove, killers[2], countermove;
```

Comment:
To get rid of the -Wreorder warning and fix travis CI use:
Move ttMove, countermove, killers[2];